homeassistant/components/amcrest/services.yaml

```diff
@@ -42,6 +42,28 @@ disable_audio:
       selector:
         text:
 
+enable_motion_detection:
+  name: Enable motion detection
+  description: Enable notification when motion is detected (must be on to enable motion recording.)
+  fields:
+    entity_id:
+      name: Entity
+      description: "Name(s) of the cameras, or 'all' for all cameras."
```

Comment:
not sure whether it is more correct to use Name(s) of the camera(s)
In any case, looks very straightforward